src/index.html

```diff
@@ -12,7 +12,39 @@
       href="styles/index.scss"
     />
```

Comment:
The `href` attribute of the `<link>` tag should point to a CSS file, not a SCSS one. The browser does not understand SCSS syntax, it understands CSS. You should compile your SCSS to CSS and then link the CSS file here.